backend/minors/PHY.yaml

```diff
@@ -0,0 +1,97 @@
+  to courses in those subjects fundamental to the student's field of interest,
+  the program requires completion of courses in quantum mechanics and encourages
+  the study of subjects such as electromagnetism, statistical mechanics, thermodynamics,
+  condensed matter physics, mathematical physics, complex analysis and partial
+  differential equations. For physics majors, the program requires courses in areas
+  such as solid-state electronics, fluid mechanics, optics/optoelectronics, control
+  theory, computers and computational methods or a variety of other applied disciplines.
+  Computer science A.B. students are required to meet the technical course requirements
+  needed to satisfy the B.S.E. degree.
+urls:
+- https://ua.princeton.edu/fields-study/minors/engineering-physics
+contacts:
+- type: Director
+  name: Waseem S. Bakr
+  email: wbakr@princeton.edu
+req_list:
+- name: Math Course
+  max_counted: 1
+  min_needed: 1
+  explanation: One 300- or 400-level math course.
+  course_list:
+  - MAT 3**
+  - MAT 4**
+- name: Five Additional Courses
+  max_counted: 1
+  min_needed: 1
+  pdfs_allowed: 1
+  explanation: |-
+    For B.S.E. students, these should be five PHY courses at the 200 level or higher,
+    including a course in quantum mechanics (PHY 208).  Of these, at least four must be
+    listed or cross-listed as physics (PHY) courses.  A fifth may be a course with strong
+    physics content from other departments such as AST or CHM, but must be approved in
+    advance by the program's committee. <br><br>
+
+    For PHY majors, these should be five engineering courses from one or more of the
+    participating engineering departments.  In order to gain exposure to the design-oriented
+    philosophy of engineering, physics students are required to take at least two of their
+    engineering courses in a coherent area of study so that a clear engineering stem can be
+    identified.
+  req_list:
+  - name: PHY Courses
+    max_counted: 1
+    min_needed: 5
+    req_list:
+    - name: PHY 208
+      max_counted: 1
+      min_needed: 1
+      course_list:
+      - PHY 208
+    - name: Other Physics Courses
+      max_counted: 4
+      min_needed: 4
+      course_list:
+      - PHY *
+  - name: BSE Courses
+    max_counted: 1
+    min_needed: 5
+    course_list:
+    - CBE *
+    - CEE *
+    - COS *
+    - ECE *
+    - MAE *
+    - ORF *
+- name: Independent Work
+  max_counted: 1
+  min_needed: 1
+  explanation: |-
+    <i>Editor's Note: Students do not enroll in any numbered course for this requirement, so hoagieplan does not check it.</i><br><br>
+
+    Close collaboration with faculty is expected. Students are required to complete,
+    with a grade of B- or better, at least one semester of independent work in an
+    appropriate area.  Physics students are encouraged to have a professor in
+    engineering serve as a reader on their senior thesis.
+
```

Comment:
IW should have no_req since student's can't enroll in numbered course for it

Reply:
Good catch